test/medley/test_runner.cljr

```diff
@@ -0,0 +1,25 @@
+(ns medley.test-runner
+  (:require [clojure.test :refer :all])
+  (:import (System.IO Directory)))
+
+(defn collect-result [results result]
+  (merge-with + results result))
+
+(defn run-ns-test [ns]
+  (require ns)
+  (test-ns ns))
+
+(defn test-results [nses]
+  (reduce collect-result (map run-ns-test nses)))
+
+(def test-nses ['medley.core-test])
+
+(defn ensure-dir [path]
+  (when-not (Directory/Exists path)
+    (Directory/CreateDirectory path)))
+
+(defn -main []
+  (let [{:keys [fail error]} (test-results test-nses)
+        failures (+ fail error)]
+    (ensure-dir "tmp")
+    (spit "tmp/status-clr.txt" failures)))
```

Comment:
Why did you choose to spit this to a file, instead of outputting to STDOUT? And why use a file in a local `tmp` directory, instead of the system temporary directory?

Reply:
Are you sure that `System.Environment/Exit` doesn't work? Could it be that cljr is interfering with it some way? Have you tried running the tests just using Clojure.Main?

Reply:
Just downloaded the CLR packages myself, and I'm getting exactly the same result you did, even for small test apps. I'm going to ask on the Clojurians slack channel to see if anyone has any ideas, as using a temporary file in this instance doesn't sit well with me.

Reply:
Awesome, gonna keep an eye on that thread. If we can get aliases working, I'd like to try the [test-runner clr port](https://github.com/dmiller/test-runner/tree/clr-port) for testing instead and remove `test-runner.cljr` altogether.

Should look something like this:

````clojure
; deps-clr.edn
{:paths ["src"]
 :aliases
 {:test
  {:extra-paths ["test"]
   :extra-deps  {io.github.dmiller/test-runner {:git/tag "v0.5.1clr" :git/sha "814e06f"}}
   :exec-fn     cognitect.test-runner.api/test
   :exec-args   {:dirs ["test"]}}}
````

````yml
# test.yml
- name: Run cljr tests
  run: cljr -X:test
````

Reply:
`test_runner.cljr` has been removed and the test command updated to `cljr X:test`